pkgs/tools/system/netdata/default.nix

```diff
@@ -53,282 +57,309 @@
   withNdsudo ? false,
   withNetfilter ? stdenv.hostPlatform.isLinux,
   withNetworkViewer ? stdenv.hostPlatform.isLinux,
+  withOtel ? true,
   withSsl ? true,
   withSystemdJournal ? stdenv.hostPlatform.isLinux,
   withSystemdUnits ? stdenv.hostPlatform.isLinux,
 }:
-stdenv.mkDerivation (finalAttrs: {
```

Comment:
Very large diff, might be good to split into a separate commit if it's a formatting change?

Reply:
I'm not sure how I can prevent this, the reason every line changed is because of the `lib.optionalAttrs (withOtel || withSystemdJournal)` at the bottom, which added indentation to every line. The change itself is not that large. I could create a PR where the cargo stuff is added and pull it out in a later PR to the optional attribute?

Reply:
Should now be easier to review